### ISSUE TITLE
Script exits with no messages, but nothing installed on anbox

### ISSUE THREAD

**Issue by sylvaticus** [geeks-r-us/anbox-playstore-installer#56]
Hi there. I am using the `install-playstore.sh` script following the instructions [here](https://www.clusterednetworks.com/blog/post/install-google-play-store-anbox).

I have anbox working (download today from a Ubuntu 20.04 desktop using the beta channel) and connected with internet (I could install F-droid and a chess program from it).

When I run however `./install-playstore.sh` in a terminal the script exits after few seconds with no messages. Nothing is installed on anbox and also no temporary folder is created.

How can I help debugging this more ?

**Comment by sylvaticus** [comment]
Work it. I add to override OPENGAPPS_RELEASEDATE with `OPENGAPPS_RELEASEDATE="20201125"` otherwise the script was giving me an empty string.
I also commented the `set -e` command, although I am not sure this was really needed.

**Comment by Niek** [comment]
The `head` part is messing up the command. This works for me:
```bash
OPENGAPPS_RELEASEDATE="$(curl -s https://api.github.com/repos/opengapps/x86_64/releases/latest | grep tag_name | grep -o "\"[0-9][0-9]*\"" | grep -o "[0-9]*")" 
```

**Comment by geeks-r-us** [comment]
Should be fixed with #55
